packages/utils/ast-utils.js

```diff
@@ -429,6 +429,62 @@ function addProperty(j, p, key, value) {
 	}
 	return;
 }
+
+/**
+ *
+ * Recursively updates an object/property to a node
+ * @param {any} j — jscodeshift API
+ * @param {Node} p - AST node
+ * @param {String} key - key of a key/val object
+ * @param {Any} value - Any type of object
+ * @returns {Node} - the updated ast
+ */
+
+function updateProperty(j, p, key, value) {
+	let valForNode;
+	if (!p) {
+		return;
+	}
+	if (Array.isArray(value)) {
+		let arrExp = j.arrayExpression([]);
+		if (safeTraverse(p, [
+			"value",
+			"value",
+		]) && p.value.value.type === "ArrayExpression") {
+			arrExp = p.value.value;
+		}
+		value.forEach(val => {
+			addProperty(j, arrExp, null, val);
+		});
+		valForNode = arrExp;
+	} else if (
```

Comment:
Could we have comment here where we explain what kind of check we're doing? That `__paths` sounds like an internal thing, so it would be better to explain why

Reply:
@ev1stensberg This check is to avoid AST nodes during the recursion, right? So, this will pass only object values from webpack config and avoid AST node objects.

Reply:
If a node has `__paths`, it's a regexp which cannot be resolved into a regular literal or identifier

Reply:
Worth leaving a comment inside the code